setup.py

```diff
@@ -257,7 +257,7 @@ def run(self):
 extras["modelcreation"] = deps_list("cookiecutter")
 
 extras["sagemaker"] = deps_list("sagemaker")
-extras["deepspeed"] = deps_list("deepspeed")
+extras["deepspeed"] = deps_list("deepspeed", "accelerate")
```

Comment:
and will need to bumpup `"accelerate>=0.9.0",` on line 100 once the accelerate releases a new version before this can be merged.